optaplanner-operator/src/k8s/school-timetabling-solver.yml

```diff
@@ -4,8 +4,17 @@ kind: Solver
 metadata:
   name: school-timetabling
 spec:
-  kafkaCluster: my-cluster
-  kafkaBootstrapServers: my-cluster-kafka-bootstrap.demo.svc.cluster.local:9092
+  amqBroker:
+    host: ex-aao-amqp-0-svc.demo.svc.cluster.local
+    port: 5672
+    managementHost: ex-aao-hdls-svc.demo.svc.cluster.local
+    usernameSecretRef:
+      key: AMQ_USER
+      name: ex-aao-credentials-secret
+    passwordSecretRef:
+      key: AMQ_PASSWORD
+      name: ex-aao-credentials-secret
   solverImage: quay.io/example/school-timetabling:latest
   scaling:
-    replicas: 1
\ No newline at end of file
+    dynamic: true
+    replicas: 3
```

Comment:
So we have dynamic scaling, but we require 3 replicas? Explain please.

Reply:
This parameter is overloaded: it means the number of replicas if we scale statically and the maximum number of replicas in case of dynamic scaling.